server/player_service.py

```diff
@@ -48,35 +51,46 @@ def clear_dirty(self):
 
     async def fetch_player_data(self, player):
         async with db.engine.acquire() as conn:
-            result = await conn.execute(
-                'SELECT mean, deviation, numGames FROM `global_rating` '
-                'WHERE id=%s', player.id)
+            sql = select([
+                avatars_list.c.url,
+                avatars_list.c.tooltip,
+                global_rating.c.mean,
+                global_rating.c.deviation,
+                global_rating.c.numGames,
+                ladder1v1_rating.c.mean,
+                ladder1v1_rating.c.deviation,
+                clan.c.tag
+            ], use_labels=True).select_from(
+                login
+                .join(global_rating)
+                .join(ladder1v1_rating)
+                .outerjoin(clan_membership)
+                .outerjoin(clan)
+                .outerjoin(avatars)
+                .outerjoin(avatars_list)
+            ).where(login.c.id == player.id)
```

Comment:
I like to write queries that way:
`select_from(login).join(...).outerjoin(...)` etc..